rollup.config.mjs

```diff
@@ -45,6 +45,17 @@ const demoSiteConfig = {
   ],
 };
 
+const templateConfig = {
```

Comment:
I added this step to rollup so we can generate a `cjs` module which we can then use during our build process